test/fuzz/BUILD

```diff
@@ -16,7 +16,7 @@ cc_library(
     visibility = ["//tools:__pkg__"],
     deps = [
         "//main/pipeline",
-        "//payload:light",
+        "//payload:payload",
```

Comment:
@DarkDimius idk if we want to test this to see how much slower this makes fuzzing